plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestingKuduServer.java

```diff
@@ -71,9 +102,23 @@ public void close()
         try (Closer closer = Closer.create()) {
             closer.register(master::stop);
             tServers.forEach(tabletServer -> closer.register(tabletServer::stop));
+            closer.register(toxiProxy::stop);
+            closer.register(network::close);
```

Comment:
Note networks have to be explicitly closed, since I re-ran these tests a bunch locally I eventually hit up against docker network limits and had to run `docker network prune`. 

This problem also occurs in several other tests:
- https://github.com/trinodb/trino/blob/master/plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java#L41
- https://github.com/trinodb/trino/blob/master/plugin/trino-druid/src/test/java/io/trino/plugin/druid/TestingDruidServer.java#L88

Reply:
Opened separate PR here: https://github.com/trinodb/trino/pull/10813/files